<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 159, 207);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="resultTextEdit">
       <property name="toolTip">
        <string>Latitude and Longitude will display here...</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>Latitude and Longitude will display here...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressLineEdit">
       <property name="toolTip">
        <string>Enter City Name</string>
       </property>
       <property name="styleSheet">
        <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
       </property>
       <property name="placeholderText">
        <string>Enter City Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fetchButton">
       <property name="toolTip">
        <string>Click to Fetch Latitude &amp; Longitude</string>
       </property>
       <property name="styleSheet">
        <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
       </property>
       <property name="text">
        <string>Fetch Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
